stable/pomerium/templates/proxy-deployment.yaml

```diff
@@ -107,17 +108,13 @@ spec:
             path: /ping
             port: https
             scheme: HTTPS
-        
-        
         volumeMounts:
         - mountPath: /etc/pomerium/
           name: policy
-     
       volumes:
       - name: policy
         configMap:
           name: {{ template "pomerium.fullname" . }}
```

Comment:
@desimone you are missing something like `{{ if not .Values.config.existingConfig }}{{ include "pomerium.fullname" . }}{{ else }}{{ .Values.config.existingConfig }}{{ end }}` here I believe